DMCompiler/DMStandard/_Standard.dm

```diff
@@ -14,6 +14,8 @@ proc/ckey(Key)
 proc/ckeyEx(Text)
 proc/clamp(Value, Low, High)
 proc/cmptext(T1)
+proc/cmptextEx(T1)
+	set opendream_unimplemented = TRUE
```

Comment:
I didn't realize this was before `Defines.dm` is imported, either move these to `UnsortedAdditions.dm` with the rest of the unimplemented procs or change these from `TRUE` to `1`